libs/community/langchain_community/llms/yuan2.py

```diff
@@ -135,6 +136,13 @@ def _call(
             input = prompt
 
         headers = {"Content-Type": "application/json"}
+        if (self.top_p or 0) > 0 and (self.top_k or 0) > 0:
+            logger.warning(
+                "top_p and top_k cannot be set simultaneously. "
+                "set top_k to 0 instead..."
+            )
+            self.top_k = 0
```

Comment:
this should be checked during initialization

Reply:
Sure, thanks for your advice. PTAL;)